### ISSUE TITLE
tests: on v2018-3-py2-opensuse-423 a declared vhost/site is not enabled

### ISSUE THREAD

**Issue by daks** [saltstack-formulas/nginx-formula#224]
With the following pillar to declare a `mysite` site:
 ```
nginx:
  ng:
    servers:
      managed:
        mysite:
          enabled: True
          config:
            - server:
              - server_name: localhost
              - listen:
                - '80 default_server'
              - index: 'index.html index.htm'
              - location ~ .htm:
                - try_files: '$uri $uri/ =404'
              - include: 'snippets/letsencrypt.conf'
```
this site is not enabled on OpenSuse42, while it is on other OS. 

File is created in `/etc/nginx/sites-available` but not in `/etc/nginx/sites-enabled`.

See https://travis-ci.com/saltstack-formulas/nginx-formula/jobs/192697675 and #222 for details

**Comment by daks** [comment]
Excerpt from highstate

```
         Name: /etc/nginx/sites-enabled/mysite - Function: file.rename - Result: Clean Started: - 09:55:33.655227 Duration: 0.663 ms
```

**Comment by myii** [comment]
Closing since fixed in #222 by fixing `map.jinja` to use `/etc/nginx/vhosts.d`.

**Comment by tacerus** [comment]
vhosts.d is now used as the enabled-dir on openSUSE, but the `include` statement in `nginx.conf` still points to `sites-enabled`. Should this be changed to lookup the grains map instead of the servers map instead?
